plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/AddToCartWithOptions.php

```diff
@@ -404,7 +405,23 @@ function ( $variation ) {
 
 			$form_html = sprintf(
 				'<form %1$s>%2$s%3$s%4$s%5$s</form>',
-				get_block_wrapper_attributes( array_merge( $wrapper_attributes, $form_attributes ) ),
+				get_block_wrapper_attributes(
+					array_merge(
+						$wrapper_attributes,
+						$form_attributes,
+						array(
+							'class' => implode(
+								' ',
+								array_filter(
+									array(
+										isset( $wrapper_attributes['class'] ) ? $wrapper_attributes['class'] : '',
+										isset( $form_attributes['class'] ) ? $form_attributes['class'] : '',
+									)
+								)
+							),
```

Comment:
Nit: Can this line be simplified to?
```suggestion
							'class' => $wrapper_attributes['class'] ?? '' . ' ' . $form_attributes['class'] ?? '',
```